json_rpc/rpcproxy.nim

```diff
@@ -0,0 +1,38 @@
+import 
```

Comment:
For new code we put `{.push raises: [Defect].}` on top of the file: see https://status-im.github.io/nim-style-guide/errors.exceptions.html

This immediately makes you see what errors your new procedures might raise and forces you to think about that.
I realize that this module is not the best example of this and that you will mostly have to annotate the calls.